src/SFA.DAS.AssessorService.Data/OrganisationStandardRepository.cs

```diff
@@ -68,7 +68,7 @@ public async Task<OrganisationStandardVersion> UpdateOrganisationStandardVersion
                         SET
                             [EffectiveFrom] = @effectiveFrom,
                             [EffectiveTo] = @effectiveTo,
-                            [DateVersionApproved] = @dateVersionApproved,
+                            [DateVersionApproved] = COALESCE(DateVersionApproved, @dateVersionApproved),
```

Comment:
@ChrisJWoodcock is DateVersionApproved used anywhere at all, if not could this be removed?